PowerAuth2ForExtensions.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+	# General information
+	s.name              = 'PowerAuth2ForExtensions'
+	s.version           = '0.1.3'
```

Comment:
don't worry about the version, it will be regenerated when the production pod will be published.